tests/llmobs/jobspec.yml

```diff
@@ -25,3 +25,8 @@ google_generativeai:
   is_snapshot: true
   env:
     SUITE_NAME: google_generativeai
+vertexai:
+  runner: riot
+  is_snapshot: true
+  env:
+    SUITE_NAME: vertexai
```

Comment:
nit: Could parallelize these too (if you think it's applicable/helpful)

Reply:
Oh interesting 🤔 It should help speed up tests since it'll split them up into X number of instances... If you didn't see any difference then no need to add, maybe there aren't enough tests to reap the benefits there